docs/release/2.1.0/release_notes.md

```diff
@@ -0,0 +1,62 @@
+Chunky 2.1.0
+============
+
+## Downloads
+
+* [Chunky Launcher v1.12.0 (win, mac, linux)](https://chunkyupdate.lemaik.de/ChunkyLauncher.jar)
+
+## Release Notes
+
+See the release notes on Github:
+
+[2.1.0-187](https://github.com/chunky-dev/chunky/releases/tag/2.1.0-187)
+
+[2.1.0-190](https://github.com/chunky-dev/chunky/releases/tag/2.1.0-190)
+
+[2.1.0-197](https://github.com/chunky-dev/chunky/releases/tag/2.1.0-197)
+
+[2.1.0-204](https://github.com/chunky-dev/chunky/releases/tag/2.1.0-204)
+
+[2.1.0-208](https://github.com/chunky-dev/chunky/releases/tag/2.1.0-208)
+
+[2.1.0-213](https://github.com/chunky-dev/chunky/releases/tag/2.1.0-213)
+
+[2.1.0](https://github.com/chunky-dev/chunky/releases/tag/2.1.0)
+
+## Changelog
+
+### New Features
+Add barrel and loom blocks
+Add support for new sign types
+Add cartography table, fletching table, smithing table, blast furnace and smoker
+Add sweet berry bush, cornflower, lily of the valley, wither rose and bamboo saplings
+Support potted variants of the new flower types
+Add honey, bee nest, beehive and honeycomb blocks
+Add the composter block
+Add cut sandstone slab and cut red sandstone slab
+Add bamboo and potted bamboo
+Add the stonecutter
+Add the grindstone
+Add the lectern
+Add the campfire
+Add lanterns
+Add dragon egg
+Add a button to copy the current frame to the clipboard
+Render cauldron water levels
```

Comment:
Should these lists be actual lists?

Reply:
Yes. But you can't even reach the 2.1 release notes from what I can gather...

Reply:
Hm, we can add links to the old releases later. Same goes for the other lists (highlights)

Reply:
I can quickly draw up a release highlights for this version anyway just so it's done

Reply:
1.14+1.15 support is basically the highlight :rofl:

Reply:
True. I mean the worlds could be loaded, just not the 1.14+1.15 blocks prior.